assets/helpers/theGrid.js

```diff
@@ -57,6 +57,8 @@ export const imageCatalogue: {
   paperLandingHeroMobile: 'c662f53516b202d56fed0a08c8f1cdbd1d132a92/0_0_924_696',
   paperDeliveryFeature: '7cd44415801daf5bcb379bb0ba348c6ed7601c01/0_0_920_820',
   paperVoucherFeature: '33ce22d8c3b65e830b91ab089f5c0670cbd6a7d9/0_0_920_820',
+  showcase: '2abea0fc492779a0f790994b9525632a0bd91ee6/0_0_2664_1288',
+  showcaseSubscribe: '2abea0fc492779a0f790994b9525632a0bd91ee6/0_0_2664_1288',
```

Comment:
Are these the same?

Reply:
Yeah, need to get real images. Hoping to merge this and refine in follow-up PRs.